internal/cmd/generate/commands/gentests/skips.go

```diff
@@ -31,6 +31,8 @@ var skipFiles = []string{
 	"ml/evaluate_data_frame.yml", // Floats as map keys
 
 	"watcher/stats/10_basic.yml", // Sets "emit_stacktraces" as string ("true"), not bool
+	
+	"search.highlight/20_fvh.yml", // bad backslash
```

Comment:
This particular change might need backporting to other branches?

Reply:
Yeah, I'll get it backported to all the relevant branches.